includes/discovery/functions.inc.php

```diff
@@ -1107,7 +1107,7 @@ function discovery_process(&$valid, $device, $sensor_type, $pre_cache)
                     }
 
                     //user_func must be applied after divisor/multiplier
-                    if (isset($user_function) && function_exists($user_function)) {
+                    if (isset($user_function) && is_callable($user_function)) {
```

Comment:
@murrant I had to change this function_exists to is_callable to handle the 'class' that comes with the name. Looks safe from the documentation.